deploy/helm/listener-operator/templates/roles.yaml

```diff
@@ -1,3 +1,46 @@
+{{ if .Capabilities.APIVersions.Has "security.openshift.io/v1" }}
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: listener-scc
```

Comment:
```suggestion
  name: stackable-listener-operator-scc
```

Reply:
I agree that the name proposed by you is better but all our other SCCs follow this pattern, e.g. `zookeeper-scc`.

Reply:
Fair.